tests/src/JIT/config/benchmark/project.json

```diff
@@ -14,6 +14,10 @@
     "System.Threading": "4.0.10",
     "System.Threading.Tasks": "4.0.10",
     "System.Threading.Tasks.Parallel": "4.0.0",
+    "System.Diagnostics.Process": "4.0.0-*",
```

Comment:
The point is that you might one day find that it doesn't build if the packages shift around.  For instance, Bruce found problems with this recently.